changelog/pending/20250922--cli-display--report-error-count-and-deploymnt-message-in-diag-sec.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: cli/display
+  description: adds a line at end of diag sec showing whether dep was success or not with error count
```

```suggestion
  description: Add a line at end of diag section showing whether deployment was success or not with error count
```

Please follow the guidelines in the document I linked.  Also this changelog is user facing, so it shouldn't use abbreviations, especially ones that can be confused with others, such as `dep`, which could mean dependency or deployment.